src/main/engine/GfxController/src/OpenGlGfxController.cpp

```diff
@@ -395,6 +418,12 @@ GfxResult<GLuint> OpenGlGfxController::bindTexture(GLuint textureId) {
     glActiveTexture(GL_TEXTURE0);
     // Binds the specific textureId to a GL_TEXTURE_2D - might only need to do once?
     glBindTexture(GL_TEXTURE_2D, textureId);
+    auto error = glGetError();
+    if (error != GL_NO_ERROR) {
```

Comment:
These copy/pasted error checks should probably be defined in a macro eventually.